azure-communication-ui/demo-app/src/calling/java/com/azure/android/communication/ui/callingcompositedemoapp/UserReportedIssueHandler.kt

```diff
@@ -42,12 +29,12 @@ class UserReportedIssueHandler : CallCompositeEventHandler<CallCompositeUserRepo
 
     private fun sendToServer(
         userMessage: String?,
-        screenshot: File?,
         callingUIVersion: String?,
         callingSDKVersion: String?,
         callHistoryRecords: List<CallCompositeCallHistoryRecord>,
         logFiles: List<File>
     ) {
+        /*
```

Comment:
why comment out this whole section?

Reply:
It's part of the tutorial/howto, work in progress.

API changes mean it needs updates, which I want to do, but not in this PR.